### DISCUSSION TITLE
[RESOLVED] Different naming convention in Java and cpp

### DISCUSSION THREAD

**Discussion by CaledoniaProject** [github/codeql#7708] [General]
In Java codeql has `class MethodAccess` and in cpp it has `FunctionCall`. The naming convention seems different, as there's no MethodAccess in cpp and no FunctionCall in java.

**Comment by smowton** [comment]
Some of this is deliberate since different languages have different standard terminology for what are fundamentally the same things. However there is definitely needless divergence and a long-term goal to reduce it where possible without causing undue difficulty to people with existing CodeQL codebases to maintain.
